keyboards/jpe230/big_knob/readme.md

```diff
@@ -0,0 +1,43 @@
+# Big Knob
+
+![big_knob](https://i.imgur.com/KLS0O7Zm.png)
```

Comment:
```suggestion
![big_knob](https://i.imgur.com/KLS0O7Zmh.png)
```

Reply:
Applying the suggested change caused the link to be invalid... Is this the expected behaviour?

Reply:
Changed suggestion to swap last `m` with `h`